icloudpd/base.py

```diff
@@ -367,6 +367,9 @@ def photos_exception_handler(ex, retries):
         photos_enumerator = tqdm(photos, **tqdm_kwargs)
         logger.set_tqdm(photos_enumerator)
 
+    if auto_delete:
```

Comment:
According to the scenario https://github.com/icloud-photos-downloader/icloud_photos_downloader/pull/182#issuecomment-714911882 same file will be in `deleted` list and in `all albums`. If we delete it first, we'll have to re-download it later, correct? If correct, then it is not an ideal situation, but still provided correct state at the end.

Reply:
Ok.  I am working on this.